### PR TITLE
End of Sprint

### PR DESCRIPTION
**Pull request by ericsorenson**
Merge master into develop
ZAP run
New diagrams
Package upgrades

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ ENV GS4JS_HOME=/usr/lib
 
 COPY package.json /home/app/package.json
 COPY package-lock.json /home/app/package-lock.json
-RUN cd /home/app && npm ci
+RUN cd /home/app && npm i
```
Comment (matthopson) [inline Dockerfile:43]: Why no clean install? Speed?

Comment (ericsorenson) [reply Dockerfile:43]: it's caused nasty bugs in the past

Comment (matthopson) [reply Dockerfile:43]: What bugs? Just curious since clean install should actually be _less_ bug-prone than npm install.

Comment (matthopson) [reply Dockerfile:43]: The only time I can think of where npm ci would put you at a disadvantage is if you aren't committing your package-lock files.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**